From: Boyuan Yang <byang@debian.org>
Date: Wed, 19 Feb 2020 16:17:59 -0500
Subject: configure.ac: Force using python3

Last-Update: 2020-05-10
---
 generate.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generate.py b/generate.py
index 231f0c6..8d226f5 100644
--- a/generate.py
+++ b/generate.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # libmypaint - The MyPaint Brush Library
 # Copyright (C) 2007-2012 Martin Renold <martinxyz@gmx.ch>
 # Copyright (C) 2012-2016 by the MyPaint Development Team.
